src/app/pages/postDetail/postDetail.component.ts

```diff
@@ -8,39 +8,8 @@ import { Router, ActivatedRoute, ParamMap } from '@angular/router';
 })
 
 export class PostDetailComponent implements OnInit {
-  post:{
-    id: number,
-    title: string,
-    username:string,
-    description: string,
-    photo: string,
-    post_status: string,
-    post_priority: string,
-    city: string,
-    state: string,
-    zip_code: string,
-    email: string,
-    customer_id:number,
-    customerId:string,
-    can_bid: boolean,
-    budget: number
-  } = {
-    id:-1,
-    title:'',
-    description:'',
-    username:'',
-    photo:'',
-    post_status:'',
-    post_priority:'',
-    city:'',
-    state:'',
-    zip_code:'',
-    email:'',
-    customer_id:null,
-    customerId:null,
-    can_bid:false,
-    budget:null
-  }
+  post: object;
```

Comment:
Again, fix this correctly.

Reply:
Jen has resolved this issue on her branch